udata/core/post/api.py

```diff
@@ -84,6 +87,10 @@ def get(self):
         if not (AdminPermission().can() and args["with_drafts"]):
             posts = posts.published()
 
+        if args["q"]:
+            phrase_query = " ".join([f'"{elem}"' for elem in args["q"].split(" ")])
+            posts = posts.search_text(phrase_query)
+
         return posts.order_by(args["sort"]).paginate(args["page"], args["page_size"])
```

Comment:
I think we can use $text_score in case of a search text as in [dataset api](https://github.com/opendatateam/udata/blob/4b8fbb046bc84aa75f983e5f9adb504b4bcb7adf/udata/core/dataset/api.py#L221).

Reply:
Hum… Not sure to do that with the default value of the param… Because if someone is sorting by a column, we may want to keep his sort, no?

Reply:
Done in 14f02a069 but not a big fan of how I implement it… But I don't see another way of doing this…

Reply:
I refactor a little bit in c840d701c